gpii/node_modules/eventLog/src/metrics.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Metrics logging.
+ * This acts upon events that are deemed interesting enough to be recorded and sent to a log server for analysis.
+ *
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The R&D leading to these results received funding from the
+ * Department of Education - Grant H421A150005 (GPII-APCP). However,
+ * these results do not necessarily represent the policy of the
+ * Department of Education, and you should not assume endorsement by the
+ * Federal Government.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion");
+
+var gpii = fluid.registerNamespace("gpii");
+
+fluid.registerNamespace("gpii.metrics");
```

Comment:
No need for a namespace if there is a component of the same name

Reply:
Removed.